portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,65 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.Arrays;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    //create array list
+    ArrayList<String> myComments = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    //greets.add("Whats up");
+    //greets.add("Hi Dese!");
+    //greets.add("Hows it going");
+
+    //convert arraylist to json
+    String json = new Gson().toJson(myComments);
+
+    //Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+  
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
+
+    // Break the text into individual words.
+    String[] words = text.split("\\s*,\\s*");
+
+    // Respond with the result.
     response.setContentType("text/html;");
-    response.getWriter().println("Hello Dese!");
+    myComments.add(text);
```

Comment:
There are different scopes into which we add objects. Request/Response scope only stays for a particular request. Did this step help. Are we trying to use mycomments object. Else we might not need to add it.

Reply:
I use the myComments  object to temporarily store the comments